src/main/java/model/order/OrderResponse.java

```diff
@@ -0,0 +1,15 @@
+package model.order;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import java.util.List;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OrderResponse {
+    private int total;
```

Comment:
Always use non-primitive data types. MAKE this change everywhere.

Reply:
ok sir